providers/PaperProvider/src/main/java/net/ess3/provider/providers/PaperCommandSender.java

```diff
@@ -25,57 +53,57 @@ public void sendMessage(ComponentLike message) {
     }
 
     @Override
-    public void sendMessage(Identified source, ComponentLike message) {
+    public void sendMessage(@NotNull Identified source, ComponentLike message) {
         sendDumbComponent(message.asComponent());
     }
 
     @Override
-    public void sendMessage(Identity source, ComponentLike message) {
+    public void sendMessage(@NotNull Identity source, ComponentLike message) {
         sendDumbComponent(message.asComponent());
     }
 
     @Override
-    public void sendMessage(Component message) {
+    public void sendMessage(@NotNull Component message) {
         sendDumbComponent(message);
     }
 
     @Override
-    public void sendMessage(Identified source, Component message) {
+    public void sendMessage(@NotNull Identified source, @NotNull Component message) {
         sendDumbComponent(message);
     }
 
     @Override
-    public void sendMessage(Identity source, Component message) {
+    public void sendMessage(@NotNull Identity source, @NotNull Component message) {
         sendDumbComponent(message);
     }
 
     @Override
-    public void sendMessage(ComponentLike message, MessageType type) {
+    public void sendMessage(ComponentLike message, @NotNull MessageType type) {
         sendDumbComponent(message.asComponent());
     }
 
     @Override
-    public void sendMessage(Identified source, ComponentLike message, MessageType type) {
+    public void sendMessage(@NotNull Identified source, ComponentLike message, @NotNull MessageType type) {
         sendDumbComponent(message.asComponent());
     }
 
     @Override
-    public void sendMessage(Identity source, ComponentLike message, MessageType type) {
+    public void sendMessage(@NotNull Identity source, ComponentLike message, @NotNull MessageType type) {
         sendDumbComponent(message.asComponent());
     }
 
     @Override
-    public void sendMessage(Component message, MessageType type) {
+    public void sendMessage(@NotNull Component message, @NotNull MessageType type) {
         sendDumbComponent(message);
     }
 
     @Override
-    public void sendMessage(Identified source, Component message, MessageType type) {
+    public void sendMessage(@NotNull Identified source, @NotNull Component message, @NotNull MessageType type) {
         sendDumbComponent(message);
     }
 
     public void sendDumbComponent(Component message) {
-        sendMessage(Bukkit.getUnsafe().legacyComponentSerializer().serialize(message));
+        sendMessage(PaperComponents.legacySectionSerializer().serialize(message));
```

Comment:
Is `PaperComponents` available on all Paper versions we support with Adventure?

Reply:
yeah i checked the 1.16.5 javadocs